Lib/py_compile.py

```diff
@@ -191,21 +194,24 @@ def main(args=None):
                 break
             filename = filename.rstrip('\n')
             try:
-                compile(filename, doraise=True)
+                compile(filename, doraise=True, quiet=0)
             except PyCompileError as error:
                 rv = 1
-                sys.stderr.write("%s\n" % error.msg)
+                if args.quiet < 2:
+                    sys.stderr.write("%s\n" % error.msg)
             except OSError as error:
                 rv = 1
-                sys.stderr.write("%s\n" % error)
+                if args.quiet < 2:
+                    sys.stderr.write("%s\n" % error)
     else:
         for filename in args:
             try:
-                compile(filename, doraise=True)
+                compile(filename, doraise=True, quiet=0)
             except PyCompileError as error:
                 # return value to indicate at least one failure
                 rv = 1
-                sys.stderr.write("%s\n" % error.msg)
+                if args.quiet < 2:
```

Comment:
`args` is a list of filenames and `py_compile` doesn't seem to use argparse. Does this throw an `AttributeError`?

Reply:
Nope, you are right on this after second thought.